website/docs/reference/global-configs/warnings.md

```diff
@@ -14,16 +14,41 @@ dbt --warn-error run
 ```
 </File>
 
-<VersionBlock firstVersion="1.4">
-
 Converting any and all warnings to errors may suit your needs perfectly, but there may be some warnings you just don't care about, and some you care about a lot.
 
 The `WARN_ERROR_OPTIONS` config gives you more granular control over _exactly which types of warnings_ are treated as errors. Warnings that should be treated as errors can be specified through `include` and/or `exclude` parameters. Warning names can be found in [dbt-core's types.py file](https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/events/types.py), where each class name that inherits from `WarnLevel` corresponds to a warning name (e.g. `AdapterDeprecationWarning`, `NoNodesForSelectionCriteria`).
 
-The `include` parameter can set to `"all"` or `"*"` to treat all warnings as exceptions, or to a list of specific warning names to treat as exceptions. When include is set to `"all"` or `"*"`, the optional `exclude` parameter can be set to exclude specifc warnings from being treated as exceptions.
+The `include` parameter can be set to `"all"` or `"*"` to treat all warnings as exceptions, or to a list of specific warning names to treat as exceptions. When include is set to `"all"` or `"*"`, the optional `exclude` parameter can be set to exclude specific warnings from being treated as exceptions.
```

Comment:
I think we need to capture part of this with the 1.7 last version as this is now called "silence"

Reply:
hey @runleonarun i'm not sure i understand - i thought `silence` was a new parameter? so are you saying the entire paragraph should be avail in 1.7 and older right bc include/exclude has changed to error/warn i think?

Reply:
Hey @mirnawong1 I'm not 100% sure, but I was going by how in the [new example](https://github.com/dbt-labs/docs.getdbt.com/pull/5434/files#diff-92205076f6f62b5d3737e562ef03e8f5c14e4ac9595174f437514528d8df369dR132-R133) we say:

>    error: # Previously called "include"
>    warn: # Previously called "exclude"

It appears as though we're reframing the language for these messages to be more specific: error, warning, silence. Before it took a mental leap to understand the logic (IMO.) If I have time, I will take a look and make some changes to this PR today.